README.md

```diff
@@ -1 +1,59 @@
-# Restaurant-page
\ No newline at end of file
+# Project Restaurant
+
+> This project is a restaurant webpage built with JavaScript by manipulating the Document object model(DOM) to create a dynamic webpage.
+
```

Comment:
Let's improve this README file by adding an image of the design in here and make sure to also add live links(demo link).

Reply:
@keneogu kindly add scripts to start your app

example: `npm start`

Reply:
@keneogu I'm here if you need help.

Reply:
@keneogu, I'm currently reviewing this PR, it would be nice if you could make the changes now so that you won't have to create another CRR.

Reply:
I think we have some image bug, the images seem not showing up:
![image](https://user-images.githubusercontent.com/31897434/116087424-bc572680-a698-11eb-84b5-10b85eabd348.png)